flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -237,6 +245,30 @@ public static void startAndInitializeCassandra() {
                 CREATE_TABLE_QUERY.replace(TABLE_NAME_VARIABLE, TABLE_NAME_PREFIX + "initial"));
     }
 
+    private static void raiseCassandraRequestsTimeouts() {
+        try {
+            final Path configurationPath = TEMPORARY_FOLDER.newFile().toPath();
+            CASSANDRA_CONTAINER.copyFileFromContainer(
+                    "/etc/cassandra/cassandra.yaml", configurationPath.toAbsolutePath().toString());
+            String configuration =
+                    new String(Files.readAllBytes(configurationPath), StandardCharsets.UTF_8);
+            String patchedConfiguration =
+                    configuration
+                            .replace("request_timeout_in_ms: 10000", "request_timeout_in_ms: 30000")
+                            .replace(
+                                    "read_request_timeout_in_ms: 5000",
+                                    "read_request_timeout_in_ms: 15000")
+                            .replace(
+                                    "write_request_timeout_in_ms: 2000",
+                                    "write_request_timeout_in_ms: 6000");
```

Comment:
I agree, it is not future proof: if we change the container version, defaults might change and the replacement would no longer work. I tried to be quick but obviously not generic enough, ok for the regex. I chose to change the defaults in the containers so that the solution could apply to all the requests but there is also the other solution of adding a timeout with each request but this seems error-prone (forget to set the timeouts in the future) to me. Would you prefer this solution ?

Reply:
I think doing it in the config is fine.

We could add safeguards that the replacements actually do something (e.g., check that the string is different) if we really want to be on the safe side.

Reply:
Fair enough, I'll add an UTest that reads the file again to test the replacements.